src/pybind/util/kaldi_table_pybind.h

```diff
@@ -65,7 +65,7 @@ void pybind_sequential_table_reader(py::module& m,
            "user can just specify the p option in the rspecifier. We make this "
            "non-const to enable things like shallow swap on the held object in "
            "situations where this would avoid making a redundant copy.",
-           py::return_value_policy::reference)
+           py::return_value_policy::copy)
```

Comment:
I am a little concerned about the waste here.
I think it would be more efficient to have the iterator call Next() *before* providing the value, but have a boolean member to see whether this is the first time __next__() was called, and if so, skip calling Next().

Reply:
I agree. I'll change it.